src/jmh/resources/nigeria_wards_external_combined.xml

```diff
@@ -0,0 +1,8220 @@
+<?xml version="1.0"?>
```

Comment:
Calling this `external_combined` seems really odd to me -- seems `internal` would be a better suffix!